trail/query/rest_views.py

```diff
@@ -96,25 +208,95 @@ def get_non_query_keys(self, queryDict, *args, **kwargs):
         # latter in __init__ of a serializer - so we need to add them in both
         serializerParams = {}
         serializerParams["keepWcs"] = querysetParams["getWcs"]
+        serializerParams["fields"] = queryDict.pop("fields", None)
 
         return queryDict, serializerParams, querysetParams
 
     def get_queryset(self, getWcs=False, *args, **kwargs):
+        """
+        Return the queryset for metatable table
+
+        Parameters
+        ----------
+        getWcs: boolean
+            whether to embed all wcs objects in the metadata query set.
+
+        Returns
+        -------
+        queryset: Django QuerySet
+            Returns the querySet for metadata table.
+
+        """
         if getWcs:
             return self.queryset.prefetch_related("wcs")
         return self.queryset
 
     def is_sky_region_query(self, qparams):
+        """
+            Helper function for describing if the query is a query base on
+            sky area.
+        """
```

Comment:
You can shorten this to `Returns True if any of the sky region parameters are found in qparams`, Otherwise you have to define all of the sections.